code/modules/admin/verbs/listahelps.dm

```diff
@@ -0,0 +1,307 @@
+/datum/adminticket
+	var/ID = "" //ID of the ticket, very important as its used to find adminhelps.
+	var/user = "" //The user of the ahelp.
+	var/uckey //The saved ckey of the adminheloing user.
+	var/admin = "N/A" //The handling admin? Like come on.
+	var/msg = "" //The adminhelp message.
+	var/resolved = "No" //Is it resolved? Its much easier to have a "Yes" or a "No", as you can directly concat it into strings making life that much easier.
+	var/permckey = "" //The perm ckey, never removed essentially.
+	var/permuser = "" //Same as above!
+	var/uID = "" //The UNIQUE id, made by putting part of the ckey and the ID together. Used internally in code.
```

Comment:
I doubt we need a UID you can use `\ref[thing]` to get a ref to the ticket for topic/href